packages/browser/src/index.bundle.feedback.ts

```diff
@@ -2,11 +2,7 @@ import { browserTracingIntegrationShim, replayIntegrationShim } from '@sentry-in
 
 export * from './index.bundle.base';
 
-export {
-  feedbackIntegration,
-  feedbackModalIntegration,
-  feedbackScreenshotIntegration,
-  getFeedback,
-} from '@sentry-internal/feedback';
+export { feedbackIntegration } from './feedback';
```

Comment:
Note that we're using `feedbackIntegration` as the default here, not `feedbackAsyncIntegration`. 
So CDN users, by default, won't have async loading. 

This is something we can talk about and change outside this PR